website/docs/d/cd_toolchain.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cd_toolchain"
+description: |-
+  Get information about cd_toolchain
+subcategory: "CD Toolchain"
+---
+
+# ibm_cd_toolchain
+
+Provides a read-only data source for cd_toolchain. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_cd_toolchain" "cd_toolchain" {
+	toolchain_id = "toolchain_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `toolchain_id` - (Required, Forces new resource, String) ID of the toolchain.
+  * Constraints: The maximum length is `36` characters. The minimum length is `36` characters. The value must match regular expression `/^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$/`.
+
+## Attribute Reference
```

Comment:
Attributes are computed variables we can't have Required/Optional for those. same for all docs
Eg * `account_id` - (String) Account ID where toolchain can be found.

Reply:
We’re using the SDK generator for all our code & doc. Is this :point_up: a bug in the generator or is there something wrong with our swagger.

Do you think we’ll have to manually update each doc ??? (I’m very hesitant to do manual updates)

Reply:
yes its a generator bug we will discuss this with the team